350Lscript.js

```diff
@@ -0,0 +1,83 @@
+//You can edit ALL of the code here
+const url = "https://api.tvmaze.com/shows/82/episodes"
+
+function getTvShowData() {
+    fetch(url)
+        .then(res => res.json())
+        .then(data => {
+            const allEpisodes = data
+            makePageForEpisodes(allEpisodes)
+        })
+}
+function makePageForEpisodes(episodeList) {
+    //parent div and child input element
+    let searchBar = document.createElement("div")
+    document.body.appendChild(searchBar);
+    searchBar.innerHTML = "<input type= 'search' placeholder = 'Live search'>";
+    document.body.insertBefore(searchBar, document.body.childNodes[0]);
+
+    const rootElem = document.getElementById("root");
+
+    for (let i = 0; i < episodeList.length; i++) {
+        let episodediv = document.createElement("div");
+        episodediv.className = "episodeDiv";
+
+        episodeName = document.createElement("h1");
+        let episodeImage = document.createElement("img");
+        episodeImage.src = episodeList[i].image.medium
+        episodeSummaryText = document.createElement("p")
+
+        //append children to parent element
+        rootElem.appendChild(episodediv);
+        episodediv.appendChild(episodeName);
+        episodediv.appendChild(episodeImage);
+        episodediv.appendChild(episodeSummaryText);
```

Comment:
Your code is displayed very neatly and in a cohesive order, makes it easy for others to read and understand!